pytorch_lightning/core/lightning.py

```diff
@@ -942,28 +942,28 @@ def configure_optimizers(self):
                     dis_sched = CosineAnnealing(discriminator_opt, T_max=10) # called every epoch
                     return [gen_opt, dis_opt], [gen_sched, dis_sched]
 
-        Some things to know
+        Some things to know:
```

Comment:
```suggestion
        .. note:: Some things to know:
```

Reply:
Is "Note:" also good instead of ".. note::" to make it consistent with "Args:", "Return" etc.?

Reply:
ok, good